docs/stdlib/datetime.rst

```diff
@@ -498,30 +495,31 @@ EdgeDB stores and outputs timezone-aware values in UTC.
       {<cal::local_date>'2022-06-20'}
 
 
-    Valid units when converting from a string (and combinations of them):
-    ``'days'``,
-    ``'weeks'``,
-    ``'months'``,
-    ``'years'``,
-    ``'decades'``,
-    ``'centuries'``,
-    ``'millennia'``.
+    When converting from a string, only the following units are valid:
+
+    - ``'days'``
+    - ``'weeks'``
+    - ``'months'``
+    - ``'years'``
+    - ``'decades'``
+    - ``'centuries'``
+    - ``'millennia'``
+
+    **Units usage**
```

Comment:
Saw one other instance of this earlier. This breaks with the style we've used to introduce examples throughout the standard library documentation (at least, the parts I've reviewed here; I have not gone through it comprehensively at this point). I would lean toward changing these to be more consistent.

```suggestion
    Units usage examples:
```

I would make this change globally to any similarly styled example introductions.